app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -662,10 +654,13 @@ public boolean onBackPressed() {
         return false;
     }
 
+    public boolean saveDocument() {
+        return saveDocument(false);
+    }
```

Comment:
suggestion: lets just have one, and always explicitly specify whether saving happens based on manual or automatic action